tablecollection ts fix for build

mfyuce opened the pull request:
if you run npm run build you get `attribute does exists` error.

### CATEGORY

Choose one

- [ X] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
npm run build fails for production grade build

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
<!--- Skip this if not applicable -->

### TEST PLAN
before just run `npm run build` after this code run again. Before it will make a distribution

### ADDITIONAL INFORMATION
<!--- Check any relevant boxes with "x" -->
<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### REVIEWERS

jenkinsci-etuzel committed:
tablecollection ts fix for build

etr2460 reviewed (commented):
Thanks for the PR! it seems like in this case, we should add typing on `column` and `cell` to avoid needing to ignore types throughout. could you make that change instead of adding the `ts-ignore`s? Thanks!

mfyuce commented:
Hi, 

Thank you for the heads up. 

Removed all `//@ts-ignore` entries and added 

```
    hidden?: boolean;
    sortable?: boolean;
    cellProps?: any;

```

to `ColumnInstance` inside `git_root/superset-frontend/src/types/react-table-config.d.ts`. Tried a npm run build and it worked the same as it did with with `ts-ignores`.

PS: As long as I see `react-table` has same names in its project like, they have `isVisible` instead of `hidden`, `canSort` instead of `sortable` etc. Why not use them in superset?

mfyuce commented:
should I send the PR to master, not 0.36 version?

ktmud commented:
This is probably already fixed in `master`. Didn't see it in CI at least.

mfyuce commented:
Yes, the diff shows that.

![image](https://user-images.githubusercontent.com/41781/86993922-df6a2580-c1ad-11ea-9084-c7d4bdd247c6.png)
